### ISSUE TITLE
Error with R 4.2.0 "Error in if (unique(sapply(x, nrow)) == length(popup)) { :    the condition has length > 1"

### ISSUE THREAD

**Issue by maelle** [r-spatial/mapview#420]
:wave: here!

We see the error

```
Error in if (unique(sapply(x, nrow)) == length(popup)) { : 
  the condition has length > 1
```

on R 4.2.0. It also turns up in the scheduled tic workflow of this repo.

**Comment by tim-salabim** [comment]
Thanks! Should be fixed now.

**Comment by maelle** [comment]
Thank you! Do you plan to submit the new version to CRAN soon-ish?

**Comment by tim-salabim** [comment]
Well, it was last released on 16.04.2022 and the CRAN checks don't show this warning. How urgent is this for you?

**Comment by maelle** [comment]
It's not urgent, I was mainly curious. I'll release watch this repo to be notified. Thanks again!
